selene/src/roblox/base.toml

```diff
@@ -338,6 +338,9 @@ type = "..."
 [[Instance.new.args]]
 type = "string" # This is changed at generate time
 
+[[Instance.new.args]]
+type = "any"

```

Comment:
This would break every case of `Instance.new("Class")` by the way, since you're not doing `required = false`.